src/js/plugin.js

```diff
@@ -312,6 +312,9 @@ class Plugin {
     if (name !== BASE_PLUGIN_NAME) {
       if (Plugin.isBasic(plugin)) {
         Player.prototype[name] = createBasicPlugin(name, plugin);
+        Object.keys(plugin).forEach(function(prop) {
+          Player.prototype[name][prop] = plugin[prop];
+        });
```

Comment:
`createBasicPlugin` is a function that [returns a function](https://github.com/videojs/video.js/blob/afa747c2fdeca36aa971f4705dc121f94549daf4/src/js/plugin.js#L91). It's a bit confusing.
But `instance` is inside the returned function, you'd want to do the property copying in the outer arrow function instead.